common/web/gesture-recognizer/src/test/auto/browser/cases/recordedCoordSequences.js

```diff
@@ -101,6 +101,7 @@ describe("Layer one - DOM -> InputSequence", function() {
 
     for(let recordingID of testRecordings) {
       it(`${recordingID}.json`, function() {
+        this.timeout(2 * testconfig.timeouts.standard);
```

Comment:
For longer-duration (as in, time-duration) pre-recorded tests, the new segmentation mechanics can lead to a number of mocked `setTimeout` calls.  On my development machine, this can lead to a need for extra time to fully emulate the sequences.

It may be possible to set a flag somewhere to disable segmentation for these tests, which would also dodge the cause.